demos/README.md

```diff
@@ -0,0 +1,35 @@
+![buildah logo](https://cdn.rawgit.com/projectatomic/buildah/master/logos/buildah-logo_large.png)
```

Comment:
I'd add a link to the top level README.md to this page.

Reply:
? github buildah home page?

Reply:
Yes please, the Buildah README.md.  We've 4 links somewhere near the top, Changelog, Installation notes, Troubleshooting Guid and Tutorials.  I'd add it to that set of links.

Reply:
Okay. I will use relative addresses e.g. `../install.md`